composer.json

```diff
@@ -2,8 +2,18 @@
     "name": "yola/yophpcs",
     "description": "Yola.com PHP Code Standard",
     "minimum-stability": "dev",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/devshep/codeception.git"
```

Comment:
Looks like your pull has been merged, 2.0 looks be be released and they also have a 2.1 tagged